### ISSUE TITLE
Stop notification once account got deactivated

### ISSUE THREAD

**Issue by kelimuttu** [mozilla/sumo#337]
Currently, when an account got deactivated, the account still receives email notifications of his/her answer on the forum. Can we disable this?

One scenario to makes sense of this need is when we have to deactivate a contributor's account due to a CPG case. The person then still receives the notification and have no way to unsubscribe from the thread since s/he loses his/her account.

**Comment by smithellis** [comment]
[WIP] Pull --> https://github.com/mozilla/kitsune/pull/5836

**Comment by emilghittasv** [comment]
As raised in slack. This issue issue is still reproducible in `stage`. Users are still receiving e-mail notifications after being deactivated via the admin panel. 

I'm moving this ticket inside the `In progress` column.

**Comment by emilghittasv** [comment]
@akatsoulas pointed out that this issue should be fixed if the user was deactivated via UI (not via admin).

I can confirm in `stage` that notifications are no longer sent to a deactivated (via UI) user. 

Tested this ticket while previously subscribing to article changes, article discussion threads and localization dashboard. After deactivating the user, e-mail notifications are no longer received when performing changes against those subscribed areas. 

I've also tried testing out subscribing to different community discussion threads and dashboards but the number of received e-mails from e-mail bandit is high and I couldn't extract & analyze if indeed the deactivate user was e-mail notified or not (but I've noticed that after re-activating the user, the previous watches were no longer available/enabled).

Moving this ticket inside the `release` column.
